.circleci/config.yml

```diff
@@ -18,3 +26,9 @@ workflows:
             gpt4all-bindings/csharp/.* run-csharp-workflow true
             gpt4all-chat/.* run-chat-workflow true
             .* run-default-workflow true
+  nightly-ui-redesign:
+    when: <<pipeline.parameters.nightly>>
+    jobs:
+      - continuation/continue:
+          configuration_path: .circleci/continue_config.yml
+          parameters: "{build-chat-offline-installers: true}"
```

Comment:
- The only parameters in continue_config.yml are `run-...-workflow`, so this won't work
- The offline installer job in continue_config.yml currently requires the hold to be manually approved, but nightly builds should be automatic

Reply:
Right, can't actually directly dispatch workflows.
Pushed some changes that should make this work as expected.

Reply:
Think it is still failing

> Error calling workflow: 'nightly-build-chat-offline-installers'
> Unknown variable(s): nightly-build-offline

Reply:
Just forgot to update the `when:` stanza in `nightly-build-chat-offline-installers`, amended.